diff --git a/node_modules/typedoc-vitepress-theme/dist/sidebars/sidebar.vitepress.js b/node_modules/typedoc-vitepress-theme/dist/sidebars/sidebar.vitepress.js
index e3ebefe..a96f47d 100644
--- a/node_modules/typedoc-vitepress-theme/dist/sidebars/sidebar.vitepress.js
+++ b/node_modules/typedoc-vitepress-theme/dist/sidebars/sidebar.vitepress.js
@@ -27,6 +27,9 @@ function getNavigationItem(navigationItem, basePath, options) {
     const hasChildren = navigationItem?.children?.length;
     return {
         text: navigationItem.title,
+        ...(Boolean(navigationItem?.url) && {
+            link: `/${basePath}/${getParsedUrl(navigationItem.url).replace(/\\/g, '/')}`,
+        }),
         ...(Boolean(navigationItem?.path) && {
             link: `/${basePath}/${getParsedUrl(navigationItem.path).replace(/\\/g, '/')}`,
         }),
